misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -311,9 +311,9 @@ private static boolean isBallerinaProject(Path path) {
     private static void writeFile(String filePath, String content) throws IOException {
         OutputStreamWriter fileWriter = null;
         try {
-            try (FileOutputStream fileStream = new FileOutputStream(new File(filePath))) {
-                fileWriter = new OutputStreamWriter(fileStream, StandardCharsets.UTF_8);
-            }
+            File newFile = new File(filePath);
+            FileOutputStream fileStream = new FileOutputStream(newFile);
```

Comment:
Is it okay not to close the stream?

Reply:
Its already there in the final block right?

Reply:
Done. There was an issue with try-with-resources closing the output stream before writing to the file. Due to that I separately closed both the output stream and the file writer inside the finally block.